release/blocker_tracker_script.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+source _liferay_common.sh
+
+JIRA_TOKEN=""
+SLACK_WEB_HOOK_URL=""
+
+function main {
+
+	lc_cd liferay-portal
+
+	local git_pull_response
+
+	git_pull_response=$(git pull origin master)
+
+	if [[ "${git_pull_response}" == *"Already up to date"* ]]
+	then
+		lc_log INFO "The master branch hasn't been updated since the last run, so no changes in the blockers have occurred."
+		exit "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	local not_yet_merged_blockers=""
+
+	local blocker_issues_keys=()
+
+	while IFS= read -r key
+	do
+		blocker_issues_keys+=("$key")
+	done < <(curl \
+		"https://liferay.atlassian.net/rest/api/3/search?jql=project%20%3D%20%22PUBLIC%20-%20Liferay%20Product%20Delivery%22%20and%20labels%20%3D%20release-blocker%20and%20status%20%21%3D%20Closed&fields=issuekey" \
+			--fail \
+			--header 'Accept: application/json' \
+			--max-time 10 \
+			--request GET \
+			--retry 3 \
+			--silent \
+			--user "jira-cloud-enterprisereleasehu@liferay.com:<$JIRA_TOKEN>" |
+			jq -r '.issues[].key'
+	)
```

Comment:
@kiwm, if you get a failure from `curl` how will you notify the developer?

Try to properly handle the possible errors coming from `curl` as you did in the `curl` which sends the Slack message :wink: